.github/workflows/build.yml

```
@@ -0,0 +1,73 @@
+
+      - name: Test Pull Requests handler
+        run: curl --location http://localhost:8007/pullRequests.php
+
+      - name: Test Pushes handler
+        run: curl --location http://localhost:8007/pushes.php
+
+      - name: Test Repositories handler
+        run: curl --location http://localhost:8007/repositories.php
+
+      - name: Test Signature handler
+        run: curl --location http://localhost:8007/signature.php
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)